src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -135,7 +135,12 @@ class SendTokenTransaction extends React.Component<Props> {
   handleDismissal = () => {
     const { navigation } = this.props;
 
-    const { isSuccess, transactionPayload } = navigation.state.params;
+    const { isSuccess, transactionPayload, goBackDismiss } = navigation.state.params;
+
+    if (goBackDismiss) {
```

Comment:
1. Naming vs behavior of the prop is quite confusing, as the name suggests that it's about triggering `goBack` navigation but in effect it also causes the rest of the code in `handleDismissal` not be invoked.

2. Should the `goBack` action be somehow dependant on `isSuccess`